src/test/testScripts/hover/testHoverLocal.test.ts

```diff
@@ -0,0 +1,143 @@
+/* eslint-disable quotes */
```

Comment:
Any linting exception must have reasoning behind it.

Reply:
Yes. There's reason behind this as well.
But seems like if implemented the way you suggested in the previous comment, that linting exception can be removed from at least this file. Thanks for the suggestion.